modules/images/regenerate-existing-images/background-process/class-perflab-background-job.php

```diff
@@ -0,0 +1,403 @@
+<?php
+/**
+ * Background job class.
+ *
+ * @since n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Class Perflab_Background_Job.
+ *
+ * Manage and run the background jobs.
+ *
+ * @since n.e.x.t
+ */
+class Perflab_Background_Job {
+	/**
+	 * Meta key for storing job name/action.
+	 *
+	 * @since n.e.x.t
+	 * @var string Job name meta key.
+	 */
```

Comment:
No need to document all these constants individually as they should be self explanatory and are mostly similar. I suggest we put them together in two groups (one of the `META_KEY` ones, the other for the `JOB_STATUS` ones), and just add a single inline comment above each group for what these are for.

This comment applies to all constants in this class.

Reply:
@ankitrox don't forget to address this here. Looks like all docblock comments should be removed for each constant and grouped.